DOCS/man/options.rst

```diff
@@ -242,6 +242,25 @@ Playback Control
     affect playback speed. Playing with an altered pitch automatically inserts
     the ``scaletempo2`` audio filter.
 
+    In a standard 12-tone scale system, octaves are separated by a factor of 2
+    whereas semitones are represented by a factor of 2^1/12. This means
+    pitches can easily be shifted up or down with a simple multiplier.
+
+    .. admonition:: Examples
+
+        ``--pitch=2``
+            Shifts the pitch up a full octave.
+        ``--pitch=0.5``
+            Shifts the pitch down an octave.
+        ``--pitch=1.498307`` (2^7/12)
+            Shifts the pitch up a perfect fifth.
+        ``--pitch=0.667420`` (1 / 2^7/12)
```

Comment:
```suggestion
        ``--pitch=0.667420`` (12/2^7)
```

Reply:
Yeah probably better.

Reply:
Should be 1 / 2^(7/12), or 2^(-7/12)

Reply:
Negative exponent is equivalent and more precise.

Reply:
I think the exponent should be in parenthesis. Costs us nothing to be less ambiguous about whether it's `2^(7/12)` or `2^7 / 12`

Reply:
Done.